Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -316,11 +319,13 @@ void MainWindow::Pause()
 
 bool MainWindow::Stop()
 {
-  if (!Core::IsRunning())
+  if (!Core::IsRunning() || m_inquiry_in_progress)
```

Comment:
@ligfx It definitely can happen, but only in some weird edge cases. I can't currently reproduce it at the first try, but it happened to me and @mimimi085181 while testing. So I do think it's a thing.